client/src/components/SecurityBanner.js

```diff
@@ -14,64 +16,121 @@ const CONNECTION_INFO_COLORS = {
   error: "red"
 }
 
+const BANNER_STYLE = {
+  height: 40
+}
+
 const css = {
   zIndex: 101,
   display: "flex",
   alignItems: "center"
 }
 
-const aCss = {
-  color: "white",
-  fontSize: "0.7em"
-}
-
 const SecurityBanner = () => {
   const { appSettings, currentUser, connection } = useContext(AppContext)
   const background = appSettings[SETTING_KEY_COLOR]
 
   return (
     <>
-      <div
-        className="banner justify-content-center justify-content-md-between"
-        style={{ ...css, background }}
+      <SecurityBannerContainer
+        className="bg-primary"
+        height={BANNER_STYLE.height}
       >
-        <>
-          <div
-            className="d-none d-md-block"
-            style={{ flexBasis: "240px" }}
-          >
-          </div>
-          <div
-            style={{ display: "flex", alignItems: "center", margin: "0 1rem" }}
-          >
-            {appSettings[SETTING_KEY_TEXT]} || {currentUser.name}{" "}
+        <VersionBox>Version : {Version}</VersionBox>
+        <SecurityTextContainer
+          bgc={background}
+          height={BANNER_STYLE.height * 0.625}
+        >
+          {appSettings[SETTING_KEY_TEXT]}
+        </SecurityTextContainer>
+        <UserBox>
+          <Dropdown as={ButtonGroup}>
             <LinkTo
               modelType="Person"
               model={currentUser}
-              style={aCss}
+              button
+              className="shadow-none"
+              variant="primary"
               showIcon={false}
             >
-              (edit)
+              <AvatarDisplayComponent
+                avatar={currentUser.avatar}
+                width={25}
+                height={25}
+              />{" "}
+              {currentUser.name}
             </LinkTo>
-          </div>
-          <VersionBox>Version : {Version}</VersionBox>
-        </>
-      </div>
+            {Settings.keycloakConfiguration.showLogoutLink && (
+              <Dropdown.Toggle
+                className="shadow-none"
+                split
+                id="dropdown-split-basic"
+              />
+            )}
+            <Dropdown.Menu>
+              <Dropdown.Item href="/api/logout">Sign out</Dropdown.Item>
```

Comment:
Try to search for something, then logout, then login again (can even be as another user): search isn't cleared. You probably need to do a `resetPages()` like before?